### ISSUE TITLE
Skipping modifiers makes the check command produce false positives

### ISSUE THREAD

**Issue by PaulRBerg** [alexfertel/bulloak#81]
## Issue Description

PR https://github.com/alexfertel/bulloak/pull/68 implemented the `--skip-modifiers` flag requested in https://github.com/alexfertel/bulloak/issues/66, however it also seems to have introduced a bug.

When modifiers are skipped, Bulloak produces false positives. It reports that the test file implements all tree branches correctly, even when it doesn't.

## Steps to Reproduce

Toggle the code below and try the following commands:

```sh
bulloak check ./recentAmountOf.tree --skip-modifiers # passes, but it shouldn't
bulloak check ./recentAmountOf.tree # fails, as it should
```

The first command passes, whereas the second fails - as it should because the test file contains `test_BlahBlahBlah` instead of `test_WhenLastUpdatedTimeInPast`.

<details><summary>Toggle to see code</summary>
<p>

`recentAmountOf.tree`:

```
RecentAmountOf_Integration_Concrete_Test
├── given null
│  └── it should revert
└── given not null
   ├── given paused
   │  └── it should return zero
   └── given not paused
      ├── when last updated time in present
      │  └── it should return zero
      └── when last updated time in past
         └── it should return the correct recent amount
```

`recentAmountOf.t.sol`:

```sol
// SPDX-License-Identifier: UNLICENSED
pragma solidity >=0.8.22;

import { Integration_Test } from "../../Integration.t.sol";

contract RecentAmountOf_Integration_Concrete_Test is Integration_Test {
    function test_RevertGiven_Null() external {
        bytes memory callData = abi.encodeCall(flow.recentAmountOf, nullStreamId);
        expectRevert_Null(callData);
    }

    function test_GivenPaused() external givenNotNull {
        flow.pause(defaultStreamId);

        // It should return zero.
        uint128 recentAmount = flow.recentAmountOf(defaultStreamId);
        assertEq(recentAmount, 0, "recent amount");
    }

    function test_WhenLastUpdatedTimeInPresent() external givenNotNull givenNotPaused {
        // Update the last time to the current block timestamp.
        updateLastTimeToBlockTimestamp(defaultStreamId);

        // It should return zero.
        uint128 recentAmount = flow.recentAmountOf(defaultStreamId);
        assertEq(recentAmount, 0, "recent amount");
    }

    function test_BlahBlahBlah() external view givenNotNull givenNotPaused {
        // It should return the correct recent amount.
        uint128 recentAmount = flow.recentAmountOf(defaultStreamId);
        assertEq(recentAmount, ONE_MONTH_STREAMED_AMOUNT, "recent amount");
    }
}
```

</p>
</details>

CC @smol-ninja, @andreivladbrg.

**Comment by alexfertel** [comment]
Hey @PaulRBerg ! Thanks for reporting. Indeed I missed a conditional when implementing this, sorry about that. I merged a fix and it should be fine now.

I plan to cut a release today, but it's a slightly bigger one than normal, so it may take me a while. In the meantime, you can install from git with the latest commit and check that it indeed works. Let me know if that is not the case 🙏 Thx!

**Comment by alexfertel** [comment]
Okay, just published v0.8.0 which includes the fix!

**Comment by PaulRBerg** [comment]
Thank you so much for the quick turnaround, @alexfertel!
